src/docs/development/add-to-app/android/add-flutter-fragment.md

```diff
@@ -94,13 +95,61 @@ public class MyActivity extends FragmentActivity {
 }
 ```
 
+Kotlin:
+```kotlin
+class MyActivity : FragmentActivity() {
+  companion object {
+    // Define a tag String to represent the FlutterFragment within this
+    // Activity's FragmentManager. This value can be whatever you'd like.
+    private const val TAG_FLUTTER_FRAGMENT = "flutter_fragment"
+  }
+
+  // Declare a local variable to reference the FlutterFragment so that you
+  // can forward calls to it later.
+  private var flutterFragment: FlutterFragment? = null
```

Comment:
When `flutterFragment` is assigned, it might be set to null, so it needs to be nullable. We could lateinit a nullable reference, but I'm not sure that would help.

Reply:
ah ok